tests/metrics/network/latency_kubernetes/latency-network.sh

```diff
@@ -18,6 +18,14 @@ function remove_tmp_file() {
 
 trap remove_tmp_file EXIT
 
+function latency_cleanup() {
+	info "Latency test cleanup"
+	kubectl delete -f latency-server.yaml
```

Comment:
I think you need to pass the full path to the yaml file.

```suggestion
	kubectl delete -f "${SCRIPT_PATH}/latency-server.yaml"
```